### PR TITLE
Add camel-lumberjack wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add Camel lumberjack wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1278,21 +1278,21 @@
...
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-lumberjack/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-lumberjack' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>mvn:io.netty/netty-common/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-buffer/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-transport/${netty-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Let's use the netty feature

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1278,21 +1278,21 @@
...
+        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-handler/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/2.0.61.Final</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Let's use the jackson feature

File: features/src/main/feature/camel-features.xml (hunk 1 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done :)

File: features/src/main/feature/camel-features.xml (hunk 2 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done :)

File: features/src/main/feature/camel-features.xml [hunk 3]
```diff
@@ -1278,21 +1278,12 @@
...
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-lumberjack/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-lumberjack' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-netty</feature>
+        <feature version='${project.version}'>camel-jackson</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: I meant the `jackson` feature not the `camel-jackson` one

File: features/src/main/feature/camel-features.xml [hunk 4]
```diff
@@ -1278,21 +1278,12 @@
...
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-lumberjack/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-lumberjack' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-netty</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: I meant the `netty` feature not the `camel-netty` one

Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: features/src/main/feature/camel-features.xml (hunk 3 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**